src/homedelivery/export.ts

```diff
@@ -234,6 +298,46 @@ async function processSubs(
 	 */
 	const streamAsString = await getStream(stream as ReadStream);
 	await upload(streamAsString, outputFileName, folder);
+
+	// Publish CloudWatch metrics
+	console.log(
+		`Publishing metrics: ${totalRowsProcessed} rows processed, ` +
+			`${missingDeliveryAgentCount} missing agents, ` +
+			`${missingAddressCount} missing addresses, ` +
+			`${missingCityCount} missing cities, ` +
+			`${missingPostcodeCount} missing postcodes, ` +
+			`${missingNameCount} missing names`,
+	);
+	await putRowsProcessed('homedelivery', totalRowsProcessed);
+
+	if (missingDeliveryAgentCount > 0) {
+		await putValidationError(
+			'MissingDeliveryAgent',
+			'homedelivery',
+			missingDeliveryAgentCount,
+		);
+	}
+	if (missingAddressCount > 0) {
+		await putValidationError(
+			'MissingAddress',
+			'homedelivery',
+			missingAddressCount,
+		);
+	}
```

Comment:
Can we make this more specific to `Street Address`?


```suggestion
	if (missingStreetAddressCount > 0) {
		await putValidationError(
			'MissingStreetAddress',
			'homedelivery',
			missingStreetAddressCount,
		);
	}
```